studio/pages/project/[ref]/logs-explorer/index.tsx

```diff
@@ -106,6 +111,11 @@ export const LogsExplorerPage: NextPageWithLayout = () => {
   }
 
   const handleDateChange = ({ to, from }: DatePickerToFrom) => {
+    if (tier?.key === 'FREE') {
```

Comment:
user may customize the time and will still be in the 1 day retention range.
Would need to extract out the logic used in the upgrader prompt to ensure that user is actually exceeding the log retention limit. can place it in the logs utils or something, since it needs to be used in the previewer as well.